src/v2/Apps/Order/Routes/Review/index.tsx

```diff
@@ -383,9 +388,19 @@ export class ReviewRoute extends Component<ReviewProps> {
               <Flex flexDirection="column">
                 <Flex flexDirection="column">
                   <ArtworkSummaryItem order={order} />
-                  <TransactionDetailsSummaryItem order={order} />
+                  <TransactionDetailsSummaryItem
+                    order={order}
+                    placeholderOverride={
+                      isMissingMetadata ? "To be confirmed*" : null
```

Comment:
In this "review" step, we can probably just pass `placeholderOverride="To be confirmed*"` given it's the last step and if there is shipping or tax is still missing it is to be confirmed. Or do I miss anything here?